examples/gallery/histograms/histogram.py

```diff
@@ -0,0 +1,34 @@
+"""
+Histogram
+---------
+The :meth:`pygmt.Figure.histogram` method can plot regular histograms.
```

Comment:
This description is rather brief. Could we expand it a little bit, e.g.:

1. How the bins can be specified using `series`.
2. Mention the types of histogram that can be plotted from changing `histtype`.